Security.md

```diff
@@ -0,0 +1,122 @@
+# Security
+
+The security model of WebAssembly has two important goals: (1) protect *users*
+from buggy or malicious modules, and (2) provide *developers* with useful
+primitives and mitigations for developing safe applications, within the
+constraints of (1).
+
+## Users
+
+Each WebAssembly module executes within a sandboxed environment that is isolated
+from the host runtime using software-based fault isolation. This implies:
+  * Applications execute independently (unless
+  [dynamic linking](DynamicLinking.md) is used), and can't escape the sandbox
+  without going through appropriate APIs (the reference monitor).
+  * Applications generally execute [deterministically](Nondeterminism.md)
+  with limited exceptions.
```

Comment:
Minor nit: it might be nice to move the Nondeterminism.md link to the "with limited exceptions" rather than the "deterministically", since Nondeterminism.md describes those exceptions.